src/Application/Adapters/WordPress/Preset.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Adapters\WordPress;
+
+use NunoMaduro\PhpInsights\Domain\Contracts\Preset as PresetContract;
+use PHP_CodeSniffer\Standards\Generic\Sniffs\PHP\ForbiddenFunctionsSniff;
+use SlevomatCodingStandard\Sniffs\TypeHints\TypeHintDeclarationSniff;
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @internal
+ */
+final class Preset implements PresetContract
+{
+    /**
+     * {@inheritDoc}
+     */
+    public static function getName(): string
+    {
+        return 'wordpress';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public static function get(): array
+    {
+        return [
+            'exclude' => [
+                'wp-admin',
+                'wp-includes',
+            ],
+            'remove'  => [
+                TypeHintDeclarationSniff::class,
+                \WooCommerce::class,
+            ],
+            'config'  => [
+                ForbiddenFunctionsSniff::class => [
+                    'forbiddenFunctions' => [
+                        'var_dump' => null,
+                    ],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public static function shouldBeApplied(array $composer): bool
+    {
+        /** @var string[] $requirements */
```

Comment:
Can you use generic Annotation here? `array<string, string>`

Reply:
I was following the other presets, should be so `array<string, string>` ?

Reply:
Yes.